routes/auth.js

```diff
@@ -0,0 +1,24 @@
+const { Router } = require("express");
+const bcrypt = require("bcryptjs");
+const User = require("../models/User.model");
+
+const saltRounds = 13;
+const router = Router();
+
+router.get("/signup", (req, res) => {
+    res.render("auth/signup");
+  });
+
+  router.get("/signup", (req, res, next) => {
+  const { username, password } = req.body;
+  bcrypt.genSalt(saltRounds)
+    .then(salt => bcrypt.hash(password, salt))
+    .then(hashedPassword => User.Login.create({username, password}))
```

Comment:
- User does not have a `Login` method. Here it should be `User.create`
- if you generated a hashed password, why not use it? instead of `{username, password}` it should be `{username, password: hashedPassword}`